### PR TITLE
Add additional virtctl commands to kubectl wrapper

### PR DESCRIPTION
**Pull request by petrkotas**
**What this PR does / why we need it**:

Migrate, start and stop are added to the kubectl
the wrapper as they often come in handy.

This addition make the use of them while developing much easier.

Signed-off-by: Petr Kotas <petr.kotas@gmail.com>

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cluster-up/kubectl.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ source ${KUBEVIRTCI_PATH}/hack/common.sh
 source ${KUBEVIRTCI_CLUSTER_PATH}/$KUBEVIRT_PROVIDER/provider.sh
 source ${KUBEVIRTCI_PATH}/hack/config.sh
 
-if [ "$1" == "console" ] || [ "$1" == "vnc" ]; then
+if [ "$1" == "console" ] || [ "$1" == "vnc" ] || [ "$1" == "migrate" ] || [ "$1" == "start" ] || [ "$1" == "stop" ]; then
```
Comment (davidvossel) [inline cluster-up/kubectl.sh]: kubectl.sh is vendored from kubevirt-ci.  you'll have to make the change in kubevirt-ci then update the `kubevirtci_git_hash` in hack/config-default.sh

Comment (petrkotas) [reply cluster-up/kubectl.sh]: Ah thanks, @davidvossel completely forgot about this.

**Comment by petrkotas** [comment]
Created https://github.com/kubevirt/kubevirtci/pull/307 to add commands.
After it is merged, will update this pr.

**Pull request merged**